changelogs/fragments/vault_syml_allow.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - vault secrets file now executes in the correct context when it is a symlink (not resolved to original file).
```

In the issue 'original path' was used to describe the symlink path rather than the realpath.
```suggestion
  - vault secrets file now executes in the correct context when it is a symlink (not resolved to canonical file).
```